region/lowernorfair/east/Mickey Mouse Room.json

```diff
@@ -798,6 +866,499 @@
+          {"and": [
+            {"enemyKill": {"enemies": [ ["Multiviola", "Multiviola"], ["Multiviola", "Multiviola", "Multiviola"] ], "explicitWeapons": ["PowerBomb"]}},
+            {"heatFrames": 500}
+          ]}
+        ]},
+        {"or": [
+          {"and": [
+            {"enemyKill": {"enemies": [["Dessgeega", "Dessgeega"]], "excludedWeapons": ["Bombs", "PseudoScrew"]}},
+            {"heatFrames": 3000}
+          ]},
+          {"and": [
+            "Plasma",
+            {"heatFrames": 650}
+          ]},
+          {"and": [
+            "Ice", "Wave", "Spazer",
+            {"heatFrames": 600}
+          ]},
+          {"and": [
+            "ScrewAttack",
+            {"heatFrames": 200}
+          ]},
+          {"and": [
+            {"enemyKill": {"enemies": [["Dessgeega", "Dessgeega"]], "explicitWeapons": ["Missile"]}},
+            {"heatFrames": 670}
+          ]},
+          {"and": [
+            {"enemyKill": {"enemies": [["Dessgeega", "Dessgeega"]], "explicitWeapons": ["Super"]}},
+            {"heatFrames": 400}
+          ]},
+          {"and": [
+            {"enemyKill": {"enemies": [["Dessgeega", "Dessgeega"]], "explicitWeapons": ["PowerBomb"]}},
+            {"heatFrames": 350}
+          ]}
+        ]},
+        {"or": [
+          {"and": [
+            {"getBlueSpeed": {"usedTiles": 23, "openEnd": 2}},
+            {"heatFrames": 200}
+          ]},
+          {"and": [
+            "h_heatProof",
+            "canDodgeWhileShooting",
+            {"enemyKill": {
+              "enemies": [["Dessgeega"], ["Dessgeega"], ["Dessgeega"], ["Multiviola"], ["Multiviola"]],
+              "excludedWeapons": ["Bombs", "PseudoScrew", "PowerBomb", "PowerBeam"]}
+            }
+          ]},
+          {"and": [
+            {"heatFrames": 400},
+            "Plasma",
+            {"or": [
+              "Wave",
+              {"heatFrames": 100}
+            ]}
+          ]},
+          {"and": [
+            {"enemyKill": {
+              "enemies": [["Dessgeega"], ["Dessgeega"], ["Dessgeega"], ["Multiviola"], ["Multiviola"]],
+              "explicitWeapons": ["Missile"]}
+            },
+            {"heatFrames": 500}
+          ]},
+          {"and": [
+            {"enemyKill": {
+              "enemies": [["Dessgeega"], ["Dessgeega"], ["Dessgeega"], ["Multiviola"], ["Multiviola"]],
+              "explicitWeapons": ["Super"]}
+            },
+            {"heatFrames": 400}
+          ]},
+          {"and": [
+            "ScrewAttack",
+            {"heatFrames": 320}
+          ]}
+        ]},
+        "h_RModeCanRefillReserves",
+        {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+        {"partialRefill": {"type": "ReserveEnergy", "limit": 20}},
+        "h_shinechargeMaxRunway",
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt",
+        {"heatFrames": 30}
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
```

Comment:
This needs to clear most of the obstacles.  Mainly CDEF so you can't ice clip.  And A to be safe.